waku/v2/node/wakunode2.nim

```diff
@@ -695,7 +695,15 @@ when isMainModule:
       waku_node_errors.inc(labelValues = ["init_db_failure"])
     else:
       sqliteDatabase = dbRes.value
-  
+    
+    # run migration 
+    info "running migration ... "
```

Comment:
This will run migration scripts without considering what tables should be created according to the rest of the config (i.e. `persistMessages` or `persistPeers`). This is totally fine, as long as the scripts themselves only run if the tables they migrate already exists (see my comment elsewhere).

Reply:
Right, I addressed this issue, now the migration is only run when the `persistMessages` is enabled.